### PR TITLE
Add link about migration to Cubing Contests & fix two inconsistencies in the results

### PR DESCRIPTION
**Pull request by mintydev789**
1. Replace the link to submit more results with a link to [Cubing Contests](https://cubingcontests.com/) with a message mentioning that the project has migrated.
2. Fix round format for Mirror Blocks at Munich Open 2010 to match the order of the results.
3. Reverse the rounds for Mirror Blocks at Kanazawa Open 2008. It is assumed that they were in the original order by mistake, since round 1 had half the number of people in finals, and it was Ao5, while finals was Bo2, which makes little sense.

### CODE REVIEW

File: biome.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "formatter": {
     "indentStyle": "space",
-    "indentSize": 2,
+    "indentWidth": 2,
```
Comment (lgarron) [inline biome.json:4]: Hmm, this shouldn't be causing a linter issue, but worth doing anyhow. I'll do that in a separate commit.

Comment (mintydev789) [reply biome.json:4]: It threw an error before, so I thought I'd try to fix it

Comment (mintydev789) [reply biome.json:4]: Should I undo this?

Comment (lgarron) [reply biome.json:4]: Actually, it looks like we can't switch to this yet, Biome is still in the process of switching.

Comment (lgarron) [reply biome.json:4]: Hmm, okay, I've made `indentWidth` work now.

**Pull request merged**